views/models/StudiesModel/index.js

```diff
@@ -0,0 +1,6 @@
+const StudiesModel = {
+  createStudyOptions: (studies) =>
```

Comment:
I don't think we need `create` in front of these functions. What do you think about a method name like `dropdownSelectOptions`?